librz/core/cmd/cmd_print.c

```diff
@@ -7365,3 +7329,33 @@ RZ_IPI RzCmdStatus rz_cmd_disassemble_summarize_block_handler(RzCore *core, int
 	disasm_strings(core, input_cmd, NULL);
 	return RZ_CMD_STATUS_OK;
 }
+
+RZ_IPI RzCmdStatus rz_cmd_base64_encode_handler(RzCore *core, int argc, const char **argv, RzOutputMode mode) {
+	ut8 *buf = calloc(1, (core->blocksize * 4) + 1);
+
+	if (!buf) {
+		RZ_LOG_ERROR("Fail to allocate memory\n");
+		return RZ_CMD_STATUS_ERROR;
+	}
+	rz_base64_encode((char *)buf, (const unsigned char *)core->block, core->blocksize);
+	rz_cons_println((const char *)buf);
+	rz_free(buf);
```

Comment:
just use `free`. rz_free was introduced at some point but it was never really used... maybe worth removing at some point, but for now i don't think it really make sense given it's not used almost anywhere.